mac_pw_pool/setup.sh

```diff
@@ -123,14 +123,17 @@ if ! mount | grep -q "$PWUSER"; then
         # CI $TEMPDIR - critical for podman-machine storage performance
         ci_tempdir="/private/tmp/ci"
         mkdir -p "$ci_tempdir"
-        sudo diskutil apfs addVolume "$local_storage_volume" APFS "ci_tempdir" -mountpoint "$ci_tempdir"
+        # Something is wrong with the machine test where a write syscall just seems stuck forever
+        # We suspect it could be related to the dynamic volume resizing, as such allocate at least 20 GB.
+        # https://github.com/containers/podman/pull/23162#issuecomment-2206082041
+        sudo diskutil apfs addVolume "$local_storage_volume" APFS "ci_tempdir" -reserve 20g -mountpoint "$ci_tempdir"
         sudo chown $PWUSER:staff "$ci_tempdir"
         sudo chmod 1770 "$ci_tempdir"
 
         # CI-user's $HOME - not critical but might as well make it fast while we're
         # adding filesystems anyway.
         ci_homedir="/Users/$PWUSER"
-        sudo diskutil apfs addVolume "$local_storage_volume" APFS "ci_homedir" -mountpoint "$ci_homedir"
+        sudo diskutil apfs addVolume "$local_storage_volume" APFS "ci_homedir" -reserve 20g -mountpoint "$ci_homedir"
```

Comment:
Q: There are hundreds of Gb available on the disk container.  Any reason not to go bigger, like 50 or 100gig?

Reply:
We can go bigger I guess, I was just going with something safe. I wasn't sure how much else the already existing volumes eat up

Reply:
It's going to be dynamic as we've found.  If this does indeed fix the problem, it would be a shame to hit it again in the future because the image grew or cleanup failed or something.  I think probably 50g isn't unreasonable.

Reply:
sure I can push this once we agree that this seem o fix the flake

Reply:
Thanks, and then feel free to merge this as well.  There's no CI for these changes, and I don't see any negative consequences so far.